elixir/Dockerfile

```diff
@@ -57,15 +45,11 @@ RUN apk add -u --no-cache libstdc++ ncurses-libs openssl curl
 
 WORKDIR /app
 
-ARG MIX_ENV="prod"
-
 ARG APPLICATION_NAME
-ARG APPLICATION_VERSION=0.0.0-dev.docker
```

Comment:
These are not used anymore so they're removed. I'll be sure the images are tagged according to #2223